### PR TITLE
Postgres: Do not copy *.sql files

### PR DESCRIPTION
**Pull request by oguzcankirmemis**
Makefile: Fix TEST environment variable to use test database

Signed-off-by: Oguzcan Kirmemis <oguzcan.kirmemis@gmail.com>

### CODE REVIEW

**Comment by oguzcankirmemis** [comment]
After merging this, [#129](https://github.com/Open-IoT-Service-Platform/oisp-frontend/pull/129) can and should be merged.

**Comment by oguzcankirmemis** [comment]
@wagmarcel , It seems like kafka is failing for some reason to start when circleci does e2e tests, the issue is also in [#129](https://github.com/Open-IoT-Service-Platform/oisp-frontend/pull/129)

File: docker-postgres/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,3 @@
 FROM postgres:9.4-alpine
 
-COPY ./oisp-frontend/public-interface/deploy/postgres/base/*.sql /docker-entrypoint-initdb.d/
```
Comment (wagmarcel) [inline docker-postgres/Dockerfile]: How is the test database created without the *.sql scripts? Is this handled then later with sequelize in frontend?

Comment (oguzcankirmemis) [reply docker-postgres/Dockerfile]: Yeah sequelize will use these scripts to create the database anyway (It was always like this, resetDB also uses these scripts)

Comment (oguzcankirmemis) [reply docker-postgres/Dockerfile]: Should I add the TEST environment variable fix back?

**Comment by wagmarcel** [comment]
Seems not to work without the patch in frontend. I can confirm that removing the copy leads to the problem.

**Comment by oguzcankirmemis** [comment]
> Seems not to work without the patch in frontend. I can confirm that removing the copy leads to the problem.

Did you try it locally?, because it worked without the frontend patch by me (At least there was no hang-up, all tests were successfull)

**Comment by wagmarcel** [comment]
But you leave the script which is trying to execute SQL scripts which are not copied, i.e. not existing. When I look at the circleci image, it happens that postgres is failing at exactly this place with an error message saying that it cannot find *.sql files. It is reproducible! 
Yes, it runs locally. I will try to understand why.

**Comment by wagmarcel** [comment]
I can meanwhile also reproduce the problem with a local build. Postgres is crashing during startup.

**Comment by oguzcankirmemis** [comment]
Oh, the problem is that create_test_database.sh is trying to find *.sql files to execute at postgres exactly at this [point](https://github.com/Open-IoT-Service-Platform/oisp-frontend/blob/ac9a5aeb6bd1d01022d48cb67f98b389a54aacfb/public-interface/scripts/database/create_test_database.sh#L26).

Since this script is in frontend, just like you said, this depends on frontend patch, and also frontend patch depends on this PR. What would you suggest to do?

**Comment by oguzcankirmemis** [comment]
If you don't want merge the two PRs simultaneously, I could make an additional PR in frontend so that create_test_database.sh only executes the for loop when there exists a *.sql file in the firstplace, this way after merging that PR it should be possible to merge this one, and at the end you can merge the frontend PR with all circleci tests passing.

**Comment by wagmarcel** [comment]
ok. Yes please make it iteratively. Merging two PR at the same time should be an exception.

**Pull request merged**